package-configuration/linux-cli/test_cli_install.py

```diff
@@ -51,10 +51,14 @@ def test_full_cli_install(self):
         finally:
             shutil.rmtree(tempdir)
 
-    def test_install_from_source(self):
+    def test_install_from_source_with_requirements(self):
         tempdir = tempfile.mkdtemp()
+        temp_requirements_file = tempfile.NamedTemporaryFile(delete=True)
+        with open(temp_requirements_file.name, 'w') as requirements_file:
```

Comment:
I delete the entire directory because it contains the virtualenv and I don't need it. What's not nice about deleting the entire directory and why would deleting a temp folder fail?